packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -67,15 +67,28 @@
 					/>
 				</div>
 			</AccordionTab>
-			<AccordionTab v-if="!isEmpty(urlArtifacts)">
+			<AccordionTab v-if="!isEmpty(urlArtifacts) && !isEmpty(githubUrls)">
 				<template #header>
 					URLs<span class="artifact-amount">({{ urlArtifacts.length }})</span>
 				</template>
 				<ul>
+					<template v-if="isEditable">
+						<li class="github-link" v-for="(url, index) in githubUrls" :key="index">
+							<Button
+								label="Import"
+								class="p-button-sm p-button-outlined"
+								icon="pi pi-cloud-download"
+								@click="openCodeBrowser(url)"
+							/>
```

Comment:
I wonder if you should create a component just for this, that will include the modal and all the GitHub integration you did, just to make things tidy and in one place.